src/pudl/helpers.py

```diff
@@ -1918,3 +1923,84 @@ def diff_wide_tables(
     return TableDiff(
         deleted=deleted, added=added, changed=changed, old_df=old, new_df=new
     )
+
+
+def create_datasette_metadata_yaml() -> str:
+    """Create datasette metadata yaml.
+
+    Returns:
+        Datasette metadata for all PUDL resources as YAML
```

Comment:
Is it just metadata for the PUDL resources? Or is it for the PUDL Resources and also all of the XBRL databases, as derived from their datapackage.json files too?

Reply:
It's both! I'll update the doc string.